source/custom-output-cli.txt

```diff
@@ -0,0 +1,250 @@
+.. _go-template-output: 
+
+=============================
+Customize the {+atlas-cli+} Output
+=============================
```

Comment:
The length of this is causing a build error. Add this and it should resolve the error
```suggestion
==================================
Customize the {+atlas-cli+} Output
==================================
```